src/libcore/convert.rs

```diff
@@ -99,28 +98,20 @@ use fmt;
 #[inline]
 pub const fn identity<T>(x: T) -> T { x }
 
-/// A cheap reference-to-reference conversion. Used to convert a value to a
-/// reference value within generic code.
-///
-/// `AsRef` is very similar to, but serves a slightly different purpose than,
-/// [`Borrow`].
+/// Used to do a cheap reference-to-reference conversion.
+/// This trait is similar to [`AsMut`] which is used for converting between mutable references.
+/// If you need to do a costly conversion it is better to implement [`From`] with type
+/// ```&T``` or write a custom function.
```

Comment:
Why use \`\`\` instead of singular \' for inlining code?

Reply:
To be truly honest, I never knew singular ` would work as well. I just changed all the code that is inlined in the text using singular ` as you proposed.

Thanks for your suggestion!

Reply:
You're welcome! A small nits: \` called [backtick or backquote](https://unix.stackexchange.com/questions/27428/what-does-backquote-backtick-mean-in-commands).

Reply:
Thanks! I didn't know that either :) I'll keep it in mind for the future. I guess there's still always a lot to learn out there :)